submissions/Olumide Ologbon/html/style.css

```diff
@@ -0,0 +1,126 @@
+.signupbox {
+    text-align: center;
+    text-size-adjust: 100px;
+    border: 1px solid red;
+    margin-left: 500px;    
+    margin-top: 50px;
+    margin-bottom: 50px;
+    background-color: rgb(235, 236, 226);
+    box-sizing: border-box;
+    display: inline-block;
+    width: 400px; 
+}        
+.head {
```

Comment:
You might want to add a line after each class.. This makes your code neater.
Well done Champ

Reply:
Okay sir. I will work on it

Reply:
Please go through again

Reply:
i have made the changes now sir.